x/poe/client/cli/tx.go

```diff
@@ -394,6 +398,49 @@ $ %s tx poe unbond 100stake --from mykey
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
 
+func NewUnjailTxCmd() *cobra.Command {
```

Comment:
But this is just for tests, right? You mention below in a comment that only the OC can do this.

If it is for actual (non-test) usage, we should make an OC proposal command. But that may be overkill with the UI already working

Reply:
The node operator can unjail his validator after the jail lock expired. The OC can unjail any time. 
This CLI command is for the node operator only.